app/assets/javascripts/discourse/tests/acceptance/composer-actions-test.js

```diff
@@ -19,11 +19,15 @@ import userFixtures from "discourse/tests/fixtures/user-fixtures";
 import { cloneJSON } from "discourse-common/lib/object";
 
 acceptance("Composer Actions", function (needs) {
-  needs.user();
+  needs.user({
+    id: 5,
+    username: "kris",
+    groups: [{ id: 14, name: "awesome_group" }],
+  });
   needs.settings({
     prioritize_username_in_ux: true,
     display_name_on_post: false,
-    enable_whispers: true,
+    enable_whispers: "14",
```

Comment:
Not sure where I'll put the test but I think having multiple group ids here will be helpful in ensuring we're mangling the data properly.